bin/restarter

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+MINIMUM_TIME_ALIVE = Integer(ENV.fetch("RESTART_MINIMUM_TIME_ALIVE", "3600"), 10)
```

Comment:
This could be a local variable instead of a constant.  That would allow you to omit the hash value below.

Reply:
seems like a good win for that, even though it is a constant in some larger sense..